src/Symfony/Component/HttpKernel/HttpCache/Store.php

```diff
@@ -25,8 +25,8 @@
 class Store implements StoreInterface
 {
     protected $root;
-    private $keyCache;
-    private $locks;
+    private \SplObjectStorage $keyCache;
+    private array $locks;
```

Comment:
we could initialize here directly instead of constructor like it's done most of the time

Reply:
Yes.